Source/WebCore/dom/EventNames.h

```diff
@@ -379,6 +379,7 @@ struct EventNames {
     bool isTouchScrollBlockingEventType(const AtomString& eventType) const;
     bool isMouseClickRelatedEventType(const AtomString& eventType) const;
     bool isMouseMoveRelatedEventType(const AtomString& eventType) const;
+    bool isTransitionAnimationEventType(const AtomString& eventType) const;
```

Comment:
I suggest to use either `isCSSTransitionEventType` (matching our internal class name) or `isTransitionEventType` (matching the IDL interface name).